liquidator/index.js

```diff
@@ -61,15 +58,31 @@ async function run() {
       });
     }
     await delay(Number(10_000));
+
+    if (!shouldPoll) {
+      break;
+    }
   }
 }
 
-module.exports = async function(callback) {
+const Poll = async function(callback) {
   try {
-    await run();
+    if (!argv.address) {
+      throw new Error("Bad input arg! Specify an `address` for the location of the expiring Multi Party.");
```

Comment:
@mrice32 @chrismaree @ptare  Do you see issue in making these argv parsing checks throw errors as opposed to exiting prematurely?

Reply:
no, I think this is perfect just like this. To be honest how we deal with inputs like the NFCT address and price will change a bit once we've established the price feed architecture so for now this makes the most sense to me.

Reply:
+1 this is perfect because the error will bubble up the user's console, and it will output a nonzero exit code in bash (meaning that scripts can recognize the failure).